rollup.config.js

```diff
@@ -61,16 +61,6 @@ const esm = [
       sourcemap: true,
     },
   },
-  {
```

Comment:
By minifying the `"module"` you drop comments so existing `/* #__PURE__ */` become useless (they disappear). 

I don't see a point in pre-minifying the "module" export. It's only going to be used in a combination with a bundler and bundlers are usually already configured to minify code for production - so by minifying upfront, you only make the content of a file more obscure and you discourage trying to debug issues related to it. I for instance very often dive into `node_modules` to debug various issues and being able to actually read the source code (well, code close to source code) and being able to modify it to test stuff is very helpful.

Reply:
That's a good point about those comments getting dropped! I was trying to figure out how to prevent those specific comments from being removed from the minified bundle but couldn't.

Also good point about not pointing to the minified bundle.

Reply:
I was trying over in https://github.com/focus-trap/tabbable/pull/72/files#diff-ff6e5f22a9c7e66987b19c0199636480R15 (though comment retention wasn't the point of that change, but still, I was thinking of this too; any way, moot point with changing the package's main/module to point to unminified bundles anyway, that retain all the comments).

Reply:
Still, we've already published the minified version of the ESM bundle, so I think we should keep this in. Now that we're pointing `module` to the non-minified version, that will work for bundlers. If someone wants the minified version for some reason, then they have it. If we remove the minified build now that it's been published, that feels like a breaking change to me, which would require a new major, and I really don't want to do another major release for some time now.